client/tx.go

```diff
@@ -285,6 +285,12 @@ type protoTxProvider interface {
 // BuildSimTx creates an unsigned tx with an empty single signature and returns
 // the encoded transaction or an error if the unsigned transaction cannot be built.
 func BuildSimTx(info *keyring.Record, txf tx.Factory, msgs ...sdk.Msg) ([]byte, error) {
+
+	// For estimating gas on evmos, let's use a terrible hack to set the gas to a fixed value
```

Comment:
I don't like terrible hacks and would like to see a slightly different solution thats a bit more general. Maybe a `min-gas` config?

Reply:
Adding a MinGasAmount parameter